sig-advocacy-and-outreach/summer-ospp/kubeSphere-tekton-integration_zh-CN.md

```diff
@@ -1,56 +1,61 @@
+Tekton 是一个云原生的 CI/CD 项目。相较于 Jenkins，Tekton 可以充分利用 Kubernetes 生态系统的优势。例如，Tekton 一个是无服务器的组件，非常容易扩展。然而，由于Tekton 缺乏用户友好的用户界面，新用户可能很难操作它。我们希望能充分利用 Tekton 的优势，使其在 KubeSphere中易于使用。
 
 ## 项目详情
 
-Currently, Jenkins is the engine of KubeSphere DevOps component. The [Pipeline controller](https://github.com/kubesphere/kubesphere/blob/master/pkg/controller/pipeline/pipeline_controller.go) is responsible for converting the CRD of pipelines to Jenkins jobs.
-Please consider the compatibility of the [CRD](https://github.com/kubesphere/kubesphere/blob/master/pkg/apis/devops/v1alpha3/pipeline_types.go). Jenkins remains to be the first choice until we have sufficient reasons to change it.
+目前，Jenkins 是 KubeSphere DevOps 组件的引擎。[Pipeline controller](https://github.com/kubesphere/kubesphere/blob/master/pkg/controller/pipeline/pipeline_controller.go) 负责将管道的CRD 转换为 Jenkins Job。
+请考虑[CRD](https://github.com/kubesphere/kubesphere/blob/master/staging/src/kubesphere.io/api/devops/v1alpha3/pipeline_types.go)的兼容性，尽可能让 Jenkins 和 Tekton 共用一个 CRD。
 
 ## 项目范围
 
-Since GSoC 2021 only has one phase, you can just finish the backend part. It means that using the CLI to achieve our goal is acceptable. For example, users can use [kubectl](https://github.com/kubernetes/kubectl) to create and trigger pipelines and also view logs of them.
+可以先完成 Tekton 的部署，然后完成 Trigger、Task、Pipeline 的管理，接着是图形化的编辑、运行、查看。在图形化阶段，可以尝试考虑兼容 CRD。
 
 ## 项目产出
 
-The potential outcomes might include:
-* Pipeline CRD compatible with Jenkins
-* Log, and the archive storage solution
-* Multi-branch pipeline support
-* A how-to document that contains the integration, usage, users case
+潜在的结果可能包括
+
+* KubeSphere 部署 Tekton 的的方案
+* 多租户下的 Tekton 集成，包括前端和后端
+* 日志，以及档案存储解决方案
+* 一个包含集成、使用、用户案例的操作文档
+* 与 Jenkins 兼容的 CRD
 
 ## 链接
 
+* https://tekton.dev/
 * http://jenkins.io/
-* https://github.com/jenkins-x
-* https://github.com/kubesphere
+* https://github.com/kubesphere/kubesphere
 
 ## 快速入门
 
-You can start with [a minimal KubeSphere](https://kubesphere.io/docs/quick-start/minimal-kubesphere-on-k8s/). [Enable the DevOps](https://kubesphere.io/docs/pluggable-components/devops/) component once KubeSphere is ready and explore KubeSphere DevOps system by yourself.
-[Install Tekton Pipelines](https://github.com/tektoncd/pipeline/blob/master/docs/install.md) first, then get familiar with various features by following [the tutorial](https://github.com/tektoncd/pipeline/blob/master/docs/tutorial.md)!
+你可以从[一个最小化的 KubeSphere](https://kubesphere.io/docs/quick-start/minimal-kubesphere-on-k8s/)开始。KubeSphere 准备好了之后，[启用 DevOps ](https://kubesphere.io/docs/pluggable-components/devops/)组件，并自己探索 KubeSphere DevOps。
```

Comment:
This doc is for Chinese students. So using [a Chinese doc](https://kubesphere.io/zh/docs/quick-start/minimal-kubesphere-on-k8s/) will be better.

Reply:
Updated